src/game/server/gamecontext.cpp

```diff
@@ -969,26 +969,34 @@ void CGameContext::SendGameMsg(int GameMsgID, int ParaI1, int ParaI2, int ParaI3
 
 void CGameContext::SendChatCommand(const CCommandManager::CCommand *pCommand, int ClientID)
 {
+	const char *pHelpText = pCommand->m_aHelpText;
 	if (ClientID == -1)
 	{
 		for (int i = 0; i < MAX_CLIENTS; i++)
 			SendChatCommand(pCommand, i);
 		return;
 	}
 
+	if (ClientID >= 0)
+	{
+		CPlayer *pPlayer = m_apPlayers[ClientID];
+		if (pPlayer)
+			pHelpText = pPlayer->Localize(pCommand->m_aHelpText);
```

Comment:
As said, i dont think we should translate help text in this PR. It's more technical and can cause issues when non-english